src/UNL/ENews/Newsletter/Story.php

```diff
@@ -70,6 +70,12 @@ function __construct($options = array())
             if (!($this->newsroom->shortname == $options['shortname'])) {
                 throw new Exception('Not a valid newsroom name.', 400);
             }
+
+            if(isset($this->newsroom->private_web_view) && $this->newsroom->private_web_view === '1' ) {
```

Comment:
formatting

Reply:
Added a space after "if". Is that what you meant with the formatting? Thanks.